index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title></title>
+    <link rel="stylesheet" href="css/website.css" />
```

Comment:
The link for the css file should be "CSS/website.css" since you named the folder "CSS" in all-capital (It is case-sensitive therefore won't recognize the file path if it was written differently).